Minichain_v0/main.py

```
@@ -0,0 +1,255 @@
+                    print("Usage: send <address> <amount>")
+                    continue
+                
+                receiver = parts[1]
+                amount = int(parts[2])
+                state = blockchain.get_state()
+                
+                tx = Transaction(address, receiver, amount, state.get_nonce(address))
+                tx.sign(wallet.encode(encoder=HexEncoder).decode())
+                
+                if mempool.add(tx, state):
+                    print(f"✅ Transaction added to mempool")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)